docs/community/contribute/checks/service-support.md

```diff
@@ -0,0 +1,49 @@
+# Add Service Support
+
+A service refers to a service by a cloud provider. This section details how to add a new service to an existing provider. All contributions need to be made to the [trivy repository](https://github.com/aquasecurity/trivy/).
+
+Before you begin, verify that the provider does not already have the service that you plan to add.
+
+Adding a new service involves two steps. The service will need a data structure to store information about the required resources, and then one or more adapters to convert input(s) into the aforementioned data structure.
+
+## Adding a new service to an existing provider
+
+To add a new service named `Bar` to a provider named `foo`, you'll need to add a new file at `pkg/providers/foo/bar/bar.go`:
+
+```
+type Bar struct {
+    // ...
+}
+```
+
+Let's say the `Bar` service manages resources called `Baz`. You'll need to add a new struct to the `Bar` struct to hold information about this resource:
+
+```
+type Bar struct {
+    // ...
+    Baz []Baz
+    // ...
+}
+
+type Baz struct {
+    types.Metadata
+	Name types.StringValue
+	Encrypted types.BoolValue
+}
```

Comment:
indentation

Reply:
ah, it looked different on VS Code, not showing the indentation